scripts/ansible/service_playbook.yml

```diff
@@ -10,4 +10,5 @@
     - name: avalanche_user
     - name: avalanche_install
     - name: avalanche_upgrade
+    - name: avalanche_staker
```

Comment:
If someone runs the `service_playbook` does this mean this will is mandatory part of the playbook run?

Currently when we start a fresh node, if the cert does not exist it will be created automatically in the user's data directory.

Does this ensure that if this is run, that the node will pick up this cert directory?

Just want to avoid support situations where people could end up messing their installations due to not understanding what they are doing...

Reply:
At the moment if someone runs `service_playbook` then yes that roll will run. I had to implement this role because avalanchego only auto-generates a certificate when `--staking-tls-{cert|key}-file` aren't specified.

The avalanche_service role does specify those, so that the location isn't forced to be a hidden directory.